submissions/maryOkosun/phpSlim/.gitignore

```diff
@@ -0,0 +1,17 @@
+# phpstorm project files
+.idea
+.env
+
+# windows thumbnail cache
+Thumbs.db
+
+# Mac DS_Store Files
+.DS_Store
+
+# composer vendor dir
+/vendor
+
+DB_HOSTNAME = "localhost"
+DB_USERNAME = "root"
+DB_PASSWORD = "mysql"
+DB_NAME = "publish_house"
```

Comment:
From line 14 - 17 shouldn't be part of the `.gitignore`

Reply:
Thanks for this. Removed them